test/fixtures/poro.rb

```diff
@@ -31,9 +36,14 @@ def id
     @attributes[:id] || @attributes['id'] || object_id
   end
 
+  def to_param
+    id.to_s
+  end
```

Comment:
Now you can use https://github.com/rails-api/active_model_serializers/blob/master/lib/active_model_serializers/model.rb !!!